app/assets/stylesheets/layouts/_footer.sass

```diff
@@ -3,7 +3,7 @@ footer
   color: $white
 
 .footer-divider
-  background-image: url('footer-colored-bar.png')
+  background-image: url('/assets/footer-colored-bar.png')
```

Comment:
You can use `image-url` here, to simplify. Check it out: https://github.com/rails/sass-rails#asset-helpers